src/Symfony/Component/Lock/Bridge/DynamoDb/Store/DynamoDbStore.php

```diff
@@ -196,7 +194,7 @@ public function exists(Key $key): bool
         $item = $existingLock->getItem();
 
         // Item not found at all
-        if ($item === []) {
+        if ([] === $item) {
```

Comment:
we should have a cs-fixer rule about comparisons to empty values...
```suggestion
        if (!$item) {
```